File: app/src/main/java/com/sdapps/formula1fy/core/utils/StringHelper.java
package com.sdapps.formula1fy.core.utils;

public class StringHelper {

    public StringHelper() {
    }

    public String getQueryFormat(final String data) {
        if (data != null) {
            return "'" + data + "'";
        } else
            return null;
    }

    public String removeWhiteSpaces(final String data){
        if(data !=null){
            return data.replaceAll("\\s", "");
        }else
            return null;
    }
}

File: app/src/main/java/com/sdapps/formula1fy/core/utils/StringHelper.kt
package com.sdapps.formula1fy.core.utils

class StringHelper {
    fun getQueryFormat(data: String?): String? {
        return if (data != null) {
            "'$data'"
        } else null
    }

    fun removeWhiteSpaces(data: String?): String? {
        return data?.replace("\\s".toRegex(), "")
    }

    fun convertValueToString(data: Int?): String? {
        return data?.toString()
    }
}
